cobalt/doc/versioning.md

```diff
@@ -1,78 +1,112 @@
 release or update is released.  It is possible that multiple updates are
 released off of the same release branch, if new bugs are discovered and fixed.
 
 ## Build ID
 
-The Cobalt Build ID represents **fine-grained** information about the state of
-the source tree for a given build. An internal Cobalt build server generates a
-monotonically increasing number for each unique set of sources that it
-sees. When an open-source release is published,
-a [`src/cobalt/build/build.id`](../build/build.id) file is included that
-specifies the build ID of that source release. The Cobalt team can reproduce the
-exact sources for a given Build ID.
+The Cobalt Build ID represents `fine-grained` information about the state of the source tree for a given build. The number is simply a sequential, monotonically increasing commit number. E.g. every cherry-picked or merged commit on the release branch increments it by one.
 
-Note that since the Build ID always increases with time, it means that the
-latest version of an older Cobalt release can have a higher Build ID than the
-latest version of a new Cobalt release. An example from above: Cobalt `4.16134`
-was produced earlier than Cobalt `3.16138`, thus has a lower Build ID.
+A [`src/cobalt/build/get_build_id.py`](../build/get_build_id.py) script is used during the build to resolve the build ID number during builds, and can be similarly used to verify it in a checkout. Here is an example to get the Build ID in 24.lts.40 tag:
+```sh
+$ git checkout 24.lts.40
+HEAD is now at ecdcfdb8e5a Cherry pick PR #3084: Add idl definition for PersistentSettingLogtraceEnable getter and setter (#3091)
+$ cobalt/build/get_build_id.py 
+1032940
 
-## Older Cobalt versioning scheme
+# An example to remove two commits
+$ git reset --hard HEAD^^
+HEAD is now at 0fab5f48a97 Cherry pick PR #2930: Add blackbox test for logTrace and watchdog violations APIs. (#3059)
 
-A Cobalt version consists of two components: The Release Number and the Build
-ID. Some real historical Cobalt version examples:
+# The Build ID is decreased by two
+$ cobalt/build/get_build_id.py 
+1032938
+```
 
-  * `2.15147`
-  * `3.16138`
-  * `4.16134`
-  * `6.18971`
+# Stable Version
 
-You get the idea. The number before the dot is the "Release Number." The number
-after the dot is the "Build ID."
+Starting with Cobalt 24, the version numbering scheme for **LTS Major and Minor
+Stable releases are now numbered by multiples of 10**. This change aligns with
+semantic versioning, adds more sensible differentiation for Stable vs
+incremental non-Stable releases, and enables the Cobalt team to deliver faster
+fixes, optimizations, and features to our mutual users. **Please only use
+Cobalt LTS releases with `Minor Version` that are multiples of 10 for YouTube
+Certification and Cobalt Device Maintenance Updates.**
 
-### Release Number
+NOTE: Pre-built Evergreen binaries are only available for Stable (non-RC)
+releases at this time.
```

Comment:
Let's also add a link to where to get the Evergreen binaries for convenience.

"NOTE: Pre-built Evergreen binaries are only available for Stable (non-RC)
releases at this time and available on the release page:
- https://github.com/youtube/cobalt/releases"

Reply:
thank you for your comment. It is updated.